tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -63,24 +77,19 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     ${rc}=  Run And Return Rc  docker %{VCH-PARAMS} exec bar-c2 ping -c3 bar-c1
     Should Be Equal As Integers  ${rc}  0
 
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -it -p 10000:80 -p 10001:80 --name webserver ${nginx}
-    Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start webserver
-    Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${output}  Error
-    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10000
-    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10001
+    Launch Container With Port Forwarding  webserver  10000  10001
+    Check Nginx Port Forwarding  10000  10001
 
     Reboot VM  %{VCH-NAME}
+    Wait For VCH Initialization  20x  10 seconds
 
-    Log To Console  Getting VCH IP ...
-    ${new-vch-ip}=  Get VM IP  %{VCH-NAME}
-    Log To Console  New VCH IP is ${new-vch-ip}
-    Replace String  %{VCH-PARAMS}  %{VCH-IP}  ${new-vch-ip}
+#    Log To Console  Getting VCH IP ...
```

Comment:
presume these are temporary while you confirm it functions?

Reply:
yes. The erroneous way of grabbing IP is present in the original test, so I'm not sure if changing this will affect anything.